test/equalStrengthRightTests.html

```diff
@@ -0,0 +1,52 @@
+<!-- Max Tarlov 8/13/19 testing abstract match constraints (Kalivoda) -->
+<html>
```

Comment:
I think it would be easier to compare the effects of the Left / Right constraints if you put them in the same tableau / test file.

Reply:
merged two files in latest commit